src/Tokenizer/Analyzer/AttributeAnalyzer.php

```diff
@@ -67,4 +68,35 @@ public static function isAttribute(Tokens $tokens, int $index): bool
 
         return 0 === $count;
     }
+
+    /**
+     * Currently support only classes.
+     *
+     * @todo: support constants, functions and properties (when there is a fixer needing it)
+     *
+     * @return array<AttributeAnalysis>
+     */
+    public static function getAttributesForElement(Tokens $tokens, int $index): array
+    {
+        if (!$tokens[$index]->isGivenKind([T_CLASS])) {
+            throw new \InvalidArgumentException("Index {$index} cannot have attributes.");
```

Comment:
let's go with more explicit msg like non-class elements not yet supported by __METHOD__.
(non-class elements can have attributes, just this analyser not yet supporting it)